.github/workflows/doc-tests.yml

```diff
@@ -0,0 +1,30 @@
+name: test-docs
```

Comment:
In the mean time https://github.com/teemtee/tmt/commit/fb6ceb6936c8a9765f16dcea87b90f4f35db8c26 happened. Is this github action still necessary?

Reply:
Yes, this is more on the CI side, and this checks various builders, e.g. [linkcheck](https://github.com/teemtee/tmt/pull/2489). I think it's unnecessary to add it to a tmt workflow, since this is more about checking the generated RTD results rather than packaged documentation that could be generated differently, but of course it can be integrated in the framework of fb6ceb6936c8a9765f16dcea87b90f4f35db8c26 if that is preferred.

About the specific builder in this PR, it's `-b html -W` which errors on any warning messages.

Reply:
wdyt @lukaszachy ?

Reply:
It's true that the `/plans/install/docs` plan basically does the same/similar thing:

https://github.com/teemtee/tmt/blob/cd0b711febb4735404ce93184554c1c376432d0b/plans/install/docs.fmf#L15-L17

Actually, it needs an update as it uses the old `make` way:

```
make: Entering directory '/home/psss/git/tmt/docs'
make: *** No rule to make target 'html'.  Stop.
make: Leaving directory '/home/psss/git/tmt/docs'
```

Reply:
Oh true. It is a bit wasteful to run it on both `epel9` and `fedora-latest-stable` especially since the actual RTD is in ubuntu. The only other difference is with navigability, which one is preferred to have it piped through tmt logger or have the plain output on GH action.

This part is subjective and for now I prefer GH action navigation because right now:
- it has more immediate startup and live-report
- testing-farm does not yet set a neutral state in order to report `warning` only, while on GH action, it is possible to make and mask experimental runs, although it is extremely clunky ([example](https://github.com/LecrisUT/tmt-cmake/blob/684a9a90618bc8fc00a9368c4a41bbfd33d8b192/.github/workflows/step_test.yaml#L40))

We can definitely convert `/plans/install/docs` to a more generic `/plans/docs` and have the individual builders there if it is preferred. Could even run it within GH action, although I still need ideas on how to handle the test output for the tmt action.

Reply:
Ok, let's drop the `/plans/install/docs` plan.